metadata-ingestion/src/datahub/ingestion/source/sql/sql_utils.py

```diff
@@ -223,3 +229,43 @@ def gen_lineage(
 
         for wu in lineage_workunits:
             yield wu
+
+
+# downgrade a schema field path
+def downgrade_v2_field_path(field_path: str) -> str:
```

Comment:
yep it'd be good to replace this with `DatasetUrn._get_simple_field_path_from_v2_field_path`